.snippets/code/tutorials/interoperability/axelar-sdk/mint-2.js

```diff
@@ -0,0 +1,7 @@
+const ORIGIN_CHAIN = 'moonbeam';
+const DESTINATION_CHAIN = 'ethereum-sepolia';
+const ORIGIN_CHAIN_ADDRESS = 'INSERT_CONTRACT_ADDRESS'; // This is the address
+// of CrossChainNFT printed in the console afte running deploy script.
+const DESTINATION_CHAIN_ADDRESS = '0x89f801C7DB23439FDdBad4f913D788F13d1d7494'; 
+// This is set to AxelarAcceptEverything on Sepolia, you can change this 
+// contract address to your own AxelarAcceptEverything.sol contract.
```

Comment:
these comments should be formatted in the same way, since they're long comments, let's put them on the line above the line they're referring to

no punctuation necessary at the end of the comments